ouroboros-network/src/Ouroboros/Cardano/Diffusion/Configuration.hs

```diff
@@ -0,0 +1,50 @@
+
+-- | One stop shop for configuring diffusion layer for upstream clients
+-- This module contains Cardano specific configuration parameters
+
+module Ouroboros.Cardano.Diffusion.Configuration
```

Comment:
For consistency sake this (and the three other modules) ought to be `Ouroboros.Network.Cardano.Diffusion`; similar to the original `Diffusion` being under `Ouroboros.Network.Diffusion` name-space.

Reply:
Good catch!